cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1193,6 +1193,44 @@
     </description>
   </property>
 
+  <property>
+    <name>router.connection.backlog</name>
+    <value>20000</value>
+    <description>Determines the connection backlog in the Router Server.</description>
+  </property>
+
+  <property>
+    <name>router.server.boss.threads</name>
+    <value>1</value>
+    <description>Deterermines the number of boss threads in the Router Server.</description>
+  </property>
+
+  <property>
+    <name>router.server.worker.threads</name>
+    <value>10</value>
+    <description>Determines the number of worker threads in the Router Server.</description>
+  </property>
+
+  <property>
+    <name>router.client.boss.threads</name>
+    <value>1</value>
+    <description>Determines the number of boss threads in the Router Client.</description>
+  </property>
+
+  <property>
+    <name>router.client.worker.threads</name>
+    <value>10</value>
+    <description>Determines the number of worker threads in in the Router Client.</description>
+  </property>
+
+  <property>
+    <name>router.connection.timeout.secs</name>
+    <value>15</value>
```

Comment:
So any keepalives need to be less than this to keep the connection open? I think that this is reasonable, as the defaults in Apache HTTPd are as "high" as 15 seconds for 1.3/2.0 and as low as 5 seconds with 2.2+... https://en.wikipedia.org/wiki/HTTP_persistent_connection#HTTP_1.1